andela/troupon
# [Bug #128054481] Fix layout of merchant signup page

#### What does this PR do?

This PR fixes the layout of logo image input form on the register merchant page.

#### Description of Task to be completed?

Style elements such that they appear in sync.

#### How should this be manually tested?

- Log in to the Troupon site with a user account.
- Visit your profile.
- Click the Merchant link found at the bottom of the navigation pane on the left.
- Click on Become a merchant.
- Verify that the page's layout is consistent and no element seems out of order.

#### What are the relevant pivotal tracker stories?

#128054481

#### Screenshots:

![screen shot 2016-08-22 at 3 45 51 pm](https://cloud.githubusercontent.com/assets/7263503/17859406/652c8922-6881-11e6-98e3-e6a2e3ebb348.png)

![screen shot 2016-08-22 at 3 46 10 pm](https://cloud.githubusercontent.com/assets/7263503/17859412/6a25b368-6881-11e6-8e08-262cc0e0c0f5.png)
